services/gateway/apis/http-version3/swagger/definitions/schemas.json

```diff
@@ -761,6 +761,10 @@
             }
         }
     },
+    "CcmSchema": {
+        "type": "object",
+        "properties": {}
```

Comment:
As we have `ccmSchema`,  can we also update swagger definition instead of empty object?

Reply:
Applied.